TelCo.ColorCoder/Program.cs

```diff
@@ -77,71 +30,44 @@ private static ColorPair GetColorFromPairNumber(int pairNumber)
-                    string.Format("Unknown Colors: {0}", pair.ToString()));
-            }
-
-            // Compute pair number and Return  
-            // (Note: +1 in compute is because pair number is 1 based, not zero)
-            return (majorIndex * colorMapMinor.Length) + (minorIndex + 1);
+            return WireColorCodes.GetPairNumberFromColor(pair.majorColor, pair.minorColor);           
         }
+
         /// <summary>
         /// Test code for the class
         /// </summary>
         /// <param name="args"></param>
         private static void Main(string[] args)
         {
             int pairNumber = 4;
-            ColorPair testPair1 = Program.GetColorFromPairNumber(pairNumber);
+            ColorPairModel testPair1 = Program.GetColorFromPairNumber(pairNumber);
             Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
-            Debug.Assert(testPair1.majorColor == Color.White);
-            Debug.Assert(testPair1.minorColor == Color.Brown);
+            Debug.Assert(testPair1.majorColor.ToString() == Color.White.Name);
+            Debug.Assert(testPair1.minorColor.ToString() == Color.Brown.Name);
 
             pairNumber = 5;
             testPair1 = Program.GetColorFromPairNumber(pairNumber);
             Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
-            Debug.Assert(testPair1.majorColor == Color.White);
-            Debug.Assert(testPair1.minorColor == Color.SlateGray);
+            Debug.Assert(testPair1.majorColor.ToString() == Color.White.Name);
+            Debug.Assert(testPair1.minorColor.ToString() == Color.SlateGray.Name);
 
             pairNumber = 23;
             testPair1 = Program.GetColorFromPairNumber(pairNumber);
             Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
-            Debug.Assert(testPair1.majorColor == Color.Violet);
-            Debug.Assert(testPair1.minorColor == Color.Green);
+            Debug.Assert(testPair1.majorColor.ToString() == Color.Violet.Name);
+            Debug.Assert(testPair1.minorColor.ToString() == Color.Green.Name);
 
-            ColorPair testPair2 = new ColorPair() { majorColor = Color.Yellow, minorColor = Color.Green };
+            ColorPair testPair2 = new ColorPair(Color.Yellow, Color.Green);
             pairNumber = Program.GetPairNumberFromColor(testPair2);
             Console.WriteLine("[In]Colors: {0}, [Out] PairNumber: {1}\n", testPair2, pairNumber);
             Debug.Assert(pairNumber == 18);
 
-            testPair2 = new ColorPair() { majorColor = Color.Red, minorColor = Color.Blue };
+            testPair2 = new ColorPair(Color.Red, Color.Blue);
             pairNumber = Program.GetPairNumberFromColor(testPair2);
             Console.WriteLine("[In]Colors: {0}, [Out] PairNumber: {1}", testPair2, pairNumber);
             Debug.Assert(pairNumber == 6);
+
+            ColorCodePrinter.PrintColorCodes();
```

Comment:
The above functionality has asserts, but `PrintColorCodes` has none. That means the entire burden of testing its output becomes manual. Can you think of some asserts, which can reduce the manual effort?